rest/processing/process.py

```diff
@@ -234,28 +267,34 @@ def get_bounds(self):
 
         return final_geometry.bounds, partial_processes_crs, mapping(final_geometry)
 
-    def get_collection_temporal_step(self):
-        load_collection_node = self.get_node_by_process_id("load_collection")
+    def get_collection_temporal_step(self, load_collection_node):
         collection = collections.get_collection(load_collection_node["arguments"]["id"])
         if not collection:
             return None
         return collection["cube:dimensions"]["t"].get("step")
 
     def get_temporal_interval(self, in_days=False):
```

Comment:
I think this should be renamed to be in plural, since it returns a list of temporal intervals

Reply:
renamed